docs/students/faqs.md

```diff
@@ -36,28 +36,28 @@ No, most jobs ads don't ask for a degree. Some positions may ask for a degree, h
 
 ### When I get stuck, what questions can I ask myself to figure out what the problem is?
 It will be hard at first to know what to ask yourself, but one way around this is to use pseudo code. Psuedo code refers to readable instructions and notes that help guide the developer through their code as they create each function or part of the code. When looking back over your code, this pseudo code can be helpful in finding out what your problem is. For example, let's say you build a calculator and one instruction you wrote for yourself is 'add two numbers together'. When you look back and realize you maybe used '-' instead of '+' between the two numbers, that instruction you left for yourself might give you a hint on what to look for. As you progress through the course, you will also learn how to 'debug' code in an effective way. Debugging simply means to fix a problem in your code.
 
-Another super effective way to get through difficulties is to explain your problem to a friend or colleague. It is a very typical issue in coding that youspend lots of time trying to solve something in vain, you become frustrated and finally cannot see the forest for the trees.  Explaining your problem to somebody else forces your mind to think the whole code through again line by line from top to bottom. This can help you understand the problem and spot where the mistake is hiding.
+Another effective way to get through difficulties is to explain your problem to a friend or colleague. It is a very typical in coding that you spend lots of time trying to solve something in vain; you become frustrated and cannot see the problem no matter how hard you look.  Explaining your problem to somebody else forces your mind to think the whole code through again line by line from top to bottom. This can help you understand the problem and spot where the mistake is hiding.
 
-If you don't have somebody on hand to talk to, talk through all the problems to your dedicated [Rubber Duck](https://www.youtube.com/watch?v=IBYrQs38gQo) who acts as your debugging mate. No duck? Don't worry! It can be anything, like a doll or even your pet. [Here is another example](https://www.youtube.com/watch?v=fdaqudiSo5c). 
+If you don't have somebody on hand to talk to, talk through all the problems with your dedicated [Rubber Duck](https://www.youtube.com/watch?v=IBYrQs38gQo) who acts as your debugging friend. No duck? Don't worry! It can be anything, like a doll or even your pet. [Here is another example](https://www.youtube.com/watch?v=fdaqudiSo5c). 
```

Comment:
There is a repeated word `talk`.
Could it be summat like this:
`If you don't have somebody on hand to talk to, speak through all with the problems...`

Reply:
Good spot! I loved that you mentioned this by the way. All of my team have 'rubber' ducks. I made them with wool because I didn't want us to use loads of plastic haha

Reply:
Btw it could be also a homework: "make your debugging doll"